test/lib/mayaUsd/render/mayaToHydra/testLookThrough.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2024 Autodesk, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import maya.cmds as cmds
+import fixturesUtils
+import mtohUtils
+import mayaUtils
+import platform
+
+class TestLookThrough(mtohUtils.MtohTestCase): #Subclassing mtohUtils.MtohTestCase to be able to call self.assertSnapshotClose
+    # MayaHydraBaseTestCase.setUpClass requirement.
+    _file = __file__
+
+    @property
+    def imageDiffFailThreshold(self):
+        return 0.01
+    
+    @property
+    def imageDiffFailPercent(self):
+        # HYDRA-837 : Wireframes seem to have a slightly different color on macOS. We'll increase the thresholds
+        # for that platform specifically for now, so we can still catch issues on other platforms.
+        if platform.system() == "Darwin":
+            return 3
+        return 0.2
+
+    #Test look through camera and maya lights (spot & area).
+    def test_LookThrough(self):
+        cmds.file(new=True, force=True)
```

Comment:
This is no longer necessary, as now a new scene will be opened before each individual test.

Reply:
good, I'll remove it